bigtable_automation/gcf/custom_instance.go

```diff
@@ -29,8 +28,6 @@ func customInternal(ctx context.Context, e GCSEvent) error {
 	bucket := os.Getenv("bucket")
 	instance := os.Getenv("instance")
 	cluster := os.Getenv("cluster")
-	nodesHigh := os.Getenv("nodesHigh")
```

Comment:
Also remove "nodesHigh" and "nodesLow" from yaml files? See: https://github.com/datacommonsorg/tools/search?q=nodesHigh

Reply:
Removed for local and prod. yamls under private still references nodes high/low for deploy.sh, so I'll leave them there for now. Thanks!